Add archive forever description (#7560)
getsentry/sentry-docs: src/docs/product/accounts/early-adopter-features/issue-archiving.mdx

You can manually archive issues that are less pressing or not applicable to you or your team. This will change their status to `Archived` and remove them from the issues list. Sentry will automatically bring an issue back to the top of the list and change its status to `Escalating` if the events in that issue significantly increase over a short period of time. There's also an option to mark an issue as `Archived` for:

- Forever
- A set period of time
- Until it occurs a set number of times
- Until a set number of users are affected

> If you archive an issue with the "Forever" option, events for that issue will still be recorded, but the issue will never escalate even if it meets escalating conditions. You can still unarchive any archived issue, including those that have been archived "Forever". All unarchived issues will show up in the "Unresolved" tab.

Reviewer: ```suggestion
If you archive an issue "Forever", events connected with that issue will continue to be recorded, but the issue will never be labeled as escalating even if it meets escalating conditions. Any archived issue (including those that have been archived forever) can be found in the "Unresolved" tab and unarchived again.
```

Author: Archived issues don't show up in unresolved tab, but unarchived issues will

Author: ```suggestion
If you archive an issue "Forever", events connected with that issue will continue to be recorded, but the issue will never be labeled as escalating even if it meets escalating conditions. You can still unarchive any archived issue, including those that have been archived "Forever". All unarchived issues will then show up in the "Unresolved" tab.
```
